tools/qdstat.in

```diff
@@ -728,6 +743,17 @@ class BusManager(Node):
         dispRows = sorter.getSorted()
         disp.formattedTable(title, heads, dispRows)
 
+        # attempt to get the qdrouterd process memory usage
+        # this may not be present on all platforms
+        try:
+            objects = self.query('org.apache.qpid.dispatch.router', ['memoryUsage'])
+            mem = objects[0].memoryUsage
+        except Exception:
+            mem = 0
+
+        if mem != 0:
+            print("\nProcess memory usage: %s" % self._mem_as_mb_str(mem))
```

Comment:
This doesn't print the MB unit specifier as in the -g table
This doesn't quote the values when --csv is in effect. I'd expect "Process memory usage:","520.2 MB" in that case.